CONTRIBUTING.md

```diff
@@ -0,0 +1,25 @@
+Contributing guidelines
+=======================
+
+Escoria is a free/libre and open source point-and-click framework for the MIT licensed game engine [Godot](http://godotengine.org/). If you're reading this document, that hopefully means you want to help us make the framework even better.
+
+Contributions can take many forms. Whether you want to help write code, suggest new features or report bugs, your contributions are welcome, and together we can make Escoria the best it can be.
+
+Making a Pull Request
+---------------------
+
+If you want to write a new feature for Escoria or help us fix a bug, the first thing you should do is to fork this project. From there you typically create a new branch to work on, make some commits, and when you're happy with the result, you push the changes to your repository on GitHub and create a pull request.
+
+Since we're a small team with no dedicated tech lead, we prefer significant, thematic commits, and will work with you to get your code in great shape before merging it. If you want help with any part of the process, don't hesitate to contact us through the channels listed in the contact section below.
+
```

Comment:
I think it might be prudent to also say something like

"If you want to introduce new functionality, please make them eg. through `ProjectSettings` or exported variables to not alter Escoria's default behavior. Therefore, take care not to include changes specific to a game you may be developing in your pull requests. Toggling  switches is the best way for game developers to make their games in the style they want.
"

Otherwise I approve of this document.

Reply:
Thanks, and yes, you are right. I'll squeeze in a paragraph about that between these two paragraphs.